rootfs/deploy.py

```diff
@@ -111,6 +111,6 @@ def download_file(tar_path):
     log_output(stream, False)
 
 print("deleting image")
-stream = client.remove_image(image=registry+'/'+imageName, force=True, noprune=False)
-if DEBUG:
+stream = client.remove_image(image=registry+'/'+imageName+':'+imageTag, force=True, noprune=False)
+if DEBUG and stream is not None:
```

Comment:
Should we make this change at line 106 and 109 as well, or is this specific to removing images?